genesis/experiment/member/member.insgoc

```diff
@@ -17,12 +17,9 @@
 package member
 
 import (
-	"github.com/insolar/insolar/core"
 	"github.com/insolar/insolar/toolkit/go/foundation"
```

Comment:
We probably should use `"github.com/insolar/insolar/logicrunner/goplugin/foundation"` here

Reply:
doesn't matter that much. we can use both at the moment